TramsDataApi/DatabaseModels/FurtherEducationEstablishments.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+
+// Code scaffolded by EF Core assumes nullable reference types (NRTs) are not used or disabled.
```

Comment:
This a generated file or can we cut this comment?

Reply:
It was a generated file, but we can remove it - the file won't (shouldn't) be regenerated.